### ISSUE TITLE
Get live view to work with data received from ODIN

### ISSUE THREAD

**Issue by MRichards99** [stfc-aeg/lpd-detector-forked#11]
Currently, there's a live view window in the FEM GUI being used for data received internally. This works however this functionality hasn't been stretched for data received from ODIN, which is the task here. I have got the live view plugin to communicate with the GUI (this has been proved by the output in the terminal and by running the contents of the `odin-live-view` repo). Need to decide how to move forward from here.

**Comment by MRichards99** [comment]
Since I will need ZeroMQ to complete this issue, I've been reading up about it this morning. I've taken some test code for the Python wrapper and run them. I've been looking about the publisher/subscriber model for this since I've been told that I'll need to use this.

**Comment by MRichards99** [comment]
This has been completed and works well! Much quicker than the internal receiver:

Below shows a comparison between running the FEM GUI using odin data and internal data, showing odin being much quicker than the previous solution. I've taken times from the first 3 runs in each GUI session and repeated this twice (taking the best time from the 2 repeats - there were no big differences between these two sets of runs on each data receiver). I've used 3 runs because odin gets a bit quicker from the 2nd run onwards. This is because the frame processor and frame receiver are reused throughout the GUI session, so no need to relaunch them for each run. I've tried to keep it as fair as possible, using the following parameters for both runs:
- 500 frames
- File writing enabled
- Live view enabled - divisor of 100 and offset of 4 (so the 4th image in every 10 frames is being displayed)
- time.time() used in both cases - I've read this isn't the most accurate for absolute timing, but I think it gives a good demonstration for relative timing (i.e. how much quicker odin is)
- Timing starts from the creation of the data receiver object to when the acquisition has completed
![image](https://user-images.githubusercontent.com/32678030/49736932-5adabc80-fc83-11e8-8590-24f86b37498d.png)
